client/verta/tests/unit_tests/conftest.py

```diff
@@ -8,29 +8,50 @@
 import pytest
 import responses
 
-from verta._internal_utils._utils import Connection, Configuration
+from verta._internal_utils._utils import Configuration, Connection
+from verta._protos.public.registry import RegistryService_pb2 as _RegistryService
+from verta.client import Client
 from verta.credentials import EmailCredentials
+from verta.registry.entities import RegisteredModelVersion
+
+
+@pytest.fixture(scope="session")
+def mock_client(mock_conn) -> Client:
+    """Return a mocked object of the Client class for use in tests"""
+    # with patch.dict(os.environ, {'VERTA_EMAIL': 'test_email@verta.ai', 'VERTA_DEV_KEY':'123test1232dev1232key123'}):
+    client = Client(ignore_conn_err=True, _connect=False)
```

Comment:
`ignore_conn_err` might not be what we want. I think it does some wild thing where it converts `5XX`s and `4XX`s into `200`s, which might mess with some tests. But maybe not?